tests/AutoStep.Tests/Projects/ProjectCompilerTests.Interaction.cs

```diff
@@ -147,9 +145,6 @@ public async Task ReCompilesInteractionFileThatHasBeenChanged()
 
             var originalCompilationresult = projFile.LastCompileResult;
 
-            // Change the file timestamp.
-            changeTime = projFile.LastCompileTime.AddMinutes(1);
-
```

Comment:
This is needed (referenced in the setup method above); however test is not failing because of a mistake in the assert below.

Reply:
The last assert should read ``projFile.LastCompileResult.Should().NotBeSameAs(originalCompilationResult)``.

Reply:
i made that change and now the test fails

Reply:
Weird; I'm looking at it now.

Reply:
Ok, this was an existing problem; I've updated the test in develop so it should be all good now.